### PR TITLE
Upgrades Build

### PR DESCRIPTION
**Pull request by juanpedromoreno**
This PR:

* Replaces sbt-org-policies
* Updates dependencies
* Migrates README to mdoc (still pending to do the same for the docs, but I'm planning to do in a separate PR).
* Updates examples to the latest versions of the libraries.

Closes #282 
Closes #255 
Closes https://github.com/47degrees/fetch/pull/285

### CODE REVIEW

File: docs/src/main/tut/docs.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ To begin, add the following dependency to your SBT build file:
 [comment]: # (Start Replace)
 
 ```scala
-"com.47deg" %% "fetch" % "1.2.2"
+"com.47deg" %% "fetch" % "@VERSION@"
```
Comment (BenFradet) [inline docs/src/main/tut/docs.md:34]: does the plugin work with tut too?

File: project/ProjectPlugin.scala [hunk 2]
```diff
@@ -65,97 +61,53 @@ object ProjectPlugin extends AutoPlugin {
...
       libraryDependencies ++= Seq(
-        "io.circe"      %% "circe-generic"       % "0.12.0-RC2",
-        "org.tpolecat"  %% s"doobie-core"        % "0.8.0-RC1",
-        "org.tpolecat"  %% s"doobie-h2"          % "0.8.0-RC1",
-        "org.tpolecat"  %% "atto-core"           % "0.7.0",
-        "org.http4s"    %% "http4s-blaze-client" % "0.21.0-M4",
-        "org.http4s"    %% "http4s-circe"        % "0.21.0-M4",
+        "io.circe"      %% "circe-generic"       % "0.13.0",
+        "org.tpolecat"  %% s"doobie-core"        % "0.9.0",
+        "org.tpolecat"  %% s"doobie-h2"          % "0.9.0",
```
Comment (BenFradet) [inline project/ProjectPlugin.scala:74]: why the string interpolation? :thinking:

File: docs/src/main/tut/docs.md (hunk 1 above)
Comment (juanpedromoreno) [reply docs/src/main/tut/docs.md:34]: Nop, but in my next PR I'll migrate from tut to mdoc.

> (still pending to do the same for the docs, but I'm planning to do in a separate PR).

File: project/ProjectPlugin.scala (hunk 2 above)
Comment (juanpedromoreno) [reply project/ProjectPlugin.scala]: It was there before 🤷‍♂️ 

Removed in https://github.com/47degrees/fetch/pull/284/commits/6341994f0649b89f999ffdf9c935b48dbd4fccf0 👍

File: docs/src/main/tut/docs.md (hunk 1 above)
Comment (BenFradet) [reply docs/src/main/tut/docs.md:34]: ah yes sorry did not see that :+1:

**Pull request merged**